src/api/app/controllers/webui/attribute_controller.rb

```diff
@@ -27,8 +36,14 @@ def edit
 
     authorize @attribute
 
-    return unless @attribute.attrib_type.value_count && (@attribute.attrib_type.value_count > @attribute.values.length)
-    (@attribute.attrib_type.value_count - @attribute.values.length).times { @attribute.values.build(attrib: @attribute) }
+    value_count = @attribute.attrib_type.value_count
+    values_length = @attribute.values.length
```

Comment:
[nitpick] why are we using `length` here? I think we should use `size` instead (the nitpick is because the code was there already)

Reply:
`length` and `size` are the same... so I won't change this.